packages/layers/src/scale-bar-layer.js

```diff
@@ -45,11 +44,13 @@ const defaultProps = {
   unit: { type: 'string', value: '', compare: true },
   size: { type: 'number', value: 1, compare: true },
   position: { type: 'string', value: 'bottom-right', compare: true },
-  length: { type: 'number', value: 0.085, compare: true }
+  length: { type: 'number', value: 0.085, compare: true },
+  snap: { type: 'boolean', value: false, compare: true }
 };
 /**
  * @typedef LayerProps
  * @type {Object}
+ * @property {boolean} snap Should the value be snapped?
```

Comment:
These comments end up in our documentation. Although they could be much better, maybe something be descriptive for this flag?

```suggestion
 * @property {boolean} snap - If true, aligns the scale bar value to predefined intervals for clearer readings, adjusting units if necessary.
```